src/inputoutput.jl

```diff
@@ -219,10 +219,11 @@ function generate_control_function(sys::AbstractODESystem, inputs = unbound_inpu
         # ps = [ps; disturbance_inputs]
     end
     inputs = map(x -> time_varying_as_func(value(x), sys), inputs)
+    disturbance_inputs = map(x -> time_varying_as_func(value(x), sys), disturbance_inputs)
```

Comment:
```suggestion
    disturbance_inputs = unwrap.(disturbance_inputs)
```

The `time_varying_as_func` stuff is hacky handling of DDEs.